### DISCUSSION TITLE
Marvel Rivals

### DISCUSSION THREAD

**Discussion by TyGreeny** [cafali/SnapKey#105] [Compatibility List Submissions]
Tested 12-14-2024.
SnapKey still does not work after public release.
Tested in both fullscreen and borderless modes if it has any relevancy.

Edit: SnapKey works but only when running as administrator.
Engine is UnrealEngine 5.3.2

**Comment by TyGreeny** [comment]
I forgot to test with administrator.
It works but only with administrator.

**Comment by minteeaa** [comment]
probably due to the way some games take priority or utilize administrator-level anticheats, this makes sense
